.pre-commit-config.yaml

```diff
@@ -6,12 +6,12 @@ repos:
     hooks:
       - id: isort
   # https://github.com/python/black#version-control-integration
-  - repo: https://github.com/python/black
+  - repo: https://github.com/psf/black
```

Comment:
seems like a bad merge?

Reply:
I think this is fine, https://github.com/psf/black *is* the official url of `black` (the old url redirects to this one). However, the version of `blackdoc` should not be changed.

Reply:
I was getting errors if I did not correct the black url

Reply:
Thanks @jukent . Looks like the address changed and potentially pre-commit doesn't redirect.